app/Libraries/ScoreCheck.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace App\Libraries;
+
+use App\Exceptions\InvariantException;
+
+class ScoreCheck
+{
+    public static function assertCompleted($score)
```

Comment:
`$score` could do with an interface

Reply:
how do one interface eloquent attributes :thinking:

Reply:
I was thinking of just an `interface` with common `@property` in the doc block for typings 🤔